gravityflow/gravityflowflowchart
# Fixed flowchart icon misplaced with Gravity Forms 2.5

## Description
[Issue# 57](https://github.com/gravityflow/backlog/issues/57)

## Testing instructions
1. Ensure the flowchart icon appears correctly on the Workflow Steps page.
2. Check the flowchart appears does not erroneously appear on the step settings page.

## Automated Test Enhancements
N/A

## Screenshots
N/A

## Documentation Changes?
N/A

## Checklist:
